src/main/java/net/dv8tion/jda/api/interactions/commands/localization/LocalizationFunction.java

```diff
@@ -27,10 +27,10 @@
  *
  * <p>
  * <b>Implementation note:</b>
- * The localization key is composed of the command/option/choice tree being walked with each component being separated by a dot
+ * The localization key is composed of the command/option/choice tree being walked, each component are separated by a dot
```

Comment:
I wrote this incorrectly. 
`The localization key is composed of the command/option/choice tree being walked, each component are separated by a dot`
This isn't proper grammar

Reply:
This still needs attention